src/index.tsx

```diff
@@ -158,73 +167,128 @@ export class App extends React.Component<{}, {
+
+        if (data["RecId"].length > 1 && !this.state.selectedRecId) {
+            return (
+                <div>
+                    Must reduce to a single record or select one from the table above.
+                    You can use RecId criteria to specify an exact record.
+                </div>
+            )
         }
-        return <div><p>Fill in answers for: {data["RecId"][0]}</p>
-            <AllQuestions 
-                onSubmit={this.onSubmitAnswers}
-                columns={_trcGlobal._info.Columns}
-            ></AllQuestions>
+
+        let index = 0;
+
+        if (this.state.selectedRecId) {
+            index = data["RecId"].findIndex((x) => x === this.state.selectedRecId);
+        }
+
+        return (
+            <div>
+                <p>Fill in answers for: {data["RecId"][index]}</p>
+                <AllQuestions
+                    onSubmit={this.onSubmitAnswers}
+                    columns={_trcGlobal._info.Columns}
+                />
             </div>
+        )
 
     }
 
     private renderBody1() {
 
-        return <div>
-            <h2>Search</h2>
-            <p>This searches for specific individuals within the sheet and will also let you update answers for them. Use <PluginLink id={"Filter"}></PluginLink> to get counts. </p>
-            <div>Enter Search criteria: (Blanks match everything)</div>
-            <div>{_trcGlobal._info.CountRecords} total records. </div>
-            <FieldInputs Names={["RecId", "First", "Last", "Address", "City", "Zip", "PrecinctName"]} onSubmit={this.onSearch}></FieldInputs>
-
-            <h3>Search Results</h3>
-            {this.state.results && <SimpleTable data={this.state.results} ></SimpleTable>}
-            {this.state.totalFound  &&  <p>Found {this.state.totalFound} total result(s).</p>}
-            {(this.state.totalFound  && this.state.totalFound > this._topN) &&  <p>Showing first {this._topN} results. Narrow the search further.</p>}
-
-            <h3>Edit answers</h3>
-            {this.renderQuestions()}
-        </div>
+        return (
+            <PluginShell
+                description={
+                    <>
+                        <p>
+                            This searches for specific individuals within the sheet and
+                            will also let you update answers for them.
+                        </p>
+                        <p>
+                            Use <PluginLink id={"Filter"}></PluginLink> to get counts.
+                        </p>
+                    </>
+                }
+                title="Search"
+            >
+                <Panel>
+                    <Grid>
+                        <Copy>
+                            <p>Enter search criteria (blanks match everything):</p>
+                        </Copy>
+                        <Copy>
+                            <p style={{ fontWeight: 600, textAlign: 'right' }}>
```

Comment:
'fontweight  :600' is too open ended - I'd like to be very prescriptive on styling (so that the consuming code has minimal styling decisions).    What do you think of something like 'bold:true', which may then translate to 'fontweight  :600'